contents/blog/how-to-pivot.md

```diff
@@ -0,0 +1,214 @@
+---
+date: 2021-01-14
+title: A story about pivots
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+PostHog has pivoted a _lot_.
+
+After 5 pivots in 6 months, we got into YCombinator last year, pivoted again whilst we were there and have now gone from the first commit to $12M raised, thousands of deployments and a team across 10 countries, in well under a year.
+
+This is that story and what we learned from it.
+
+##You'll feel silly
+
+It goes something like this:
+
+1. Convince yourself then your family, friends and colleagues you have some great idea.
+1. Quit your job.
+1. Build it. Listen to the soundtrack from [The Social Network](https://www.imdb.com/title/tt1285016/) way too much.
+1. Everyone thinks your thing is terrible. Hopefully you realize.
+
+The nature of a startup is that you have to [talk to users](https://www.ycombinator.com/library/6g-how-to-talk-to-users). Or so we've heard.
+
+My sole focus for weeks on end was just to get meetings with people that we felt may have the same problem we were trying to solve.
+
+## It got good eventually, right?
+
+In 9 months, we built 6 products and did more than 100 meetings with potential users.
+
+The range of ideas we tried to solve looks broad, but the thing that connected all of them was that we focused on problems we felt we'd experienced in our previous professional lives.
+
+So, what did we build?
+
+### 1. Sales Territory Management Tool
+
+At one stage in my life, I was the VP Sales at an enterprise software company. On paper, it looked like a glamorous job - I used to fly around the world with the sales team, to go to important meetings with huge banks, in fancy skyscrapers, like the [ICC in Hong Kong](https://en.wikipedia.org/wiki/International_Commerce_Centre):
+
+![International Commerce Center - a big skyscraper in Hong Kong](../images/blog/pivoting/international-commerce-centre.jpg)
+
+I vividly remember the realization that over 90% of your sales pipeline, roughly, will never close. Even if you're really, really good at sales.
+
+The _majority_ of your time is spent getting nowhere. All those hotels, flights, calls, fastidiously wearing a suit in inappropriately warm weather - very few of those things actually result in anything.
+
+Sidenote: this is why [product led growth](../handbook/growth/strategy) is so much better.
+
+As deals get older, after a while, the probability they close drops off a cliff. If you're not getting anywhere with a potential customer after a few weeks or months of trying, it's going to get worse not better. Yet systems that are the core products of [\$17.1Bn revenue companies](https://en.wikipedia.org/wiki/Salesforce) by default come with a manually selected arbitrary number for the percentage probability.
+
+There comes a certain time, when it's rational to stop focussing on a given customer and to move on - a fresh customer is more likely to close than one that has been repeatedly ignoring you.
```

Comment:
"focusing" since we use "focused"

The bot corrects "focussed" but not "focussing" - will update it